examples/deepcopy-gen/generators/deepcopy.go

```diff
@@ -471,6 +475,10 @@ func (g *genDeepCopy) needsGeneration(t *types.Type) bool {
 	return true
 }
 
+func tagValueIsTrue(val string) bool {
```

Comment:
This seems like something that will surprise me later.  Can you at least comment why "" is considered true here?  Better, rename it to be clear